tools/hicup/hicup_mapper.xml

```diff
@@ -44,11 +69,11 @@
     </outputs>
      <tests>
         <test>
-            <param name="input_first_sequence" value="dataset1.trunc.fastq" ftype="fastq"/>
-            <param name="input_second_sequence" value="dataset2.trunc.fastq" ftype="fastq"/>
+            <param name="input_first_sequence" value="dataset1.trunc.fastq.gz" ftype="fastq.gz"/>
+            <param name="input_second_sequence" value="dataset2.trunc.fastq.gz" ftype="fastq.gz"/>
```

Comment:
These are binaries and we can not diff them. If we want to keep the compressed version we should use something like `sim_size`.

Reply:
I do not know sim_size...

Reply:
Have a look here: https://docs.galaxyproject.org/en/latest/dev/schema.html#id84